modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -235,7 +235,9 @@ def check_file_number_exists!
           ssn = target_veteran.ssn
 
           begin
+            ClaimsApi::Logger.log('poa', detail: 'Starting bgs-ext service build')
```

Comment:
Time stamps might be helpful here.

Reply:
timestamps display free of charge in Grafana.